### PR TITLE
Correct spelling

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by MarcelRobitaille — APPROVED** [review]

**Review by christianlupus — APPROVED** [review]
Looks good to me.

File: lib/Service/RecipeService.php [hunk 1]
```diff
@@ -1100,7 +1100,7 @@ public function parseRecipeFile($file) {
 	public function getRecipeImageFileByFolderId($id, $size = 'thumb'): File {
 		$recipe_folders = $this->root->getById($id);
 		if (count($recipe_folders) < 1) {
-			throw new Exception($this->il10n->t('Recipe with id %d not found.', [$id]));
+			throw new Exception($this->il10n->t('Recipe with ID %d not found.', [$id]));
```
Comment (christianlupus) [inline lib/Service/RecipeService.php:1103]: @Valdnet i have one question regarding this change: is it only syntactical to have a better English writing or is this one a technical thing (do we have a duplicate strong here somewhere)

**Comment by christianlupus** [comment]
If you do not mind, i will append a note to the Changelog. Your are free to do this yourself, i will need some time to get to my machine.

**Comment by Valdnet** [comment]
> If you do not mind, i will append a note to the Changelog.

No problem, you can add 😉.

File: lib/Service/RecipeService.php (hunk 1 above)
Comment (Valdnet) [reply lib/Service/RecipeService.php:1103]: If the ID is a direct identifier and points to an individual, the item or item should be in uppercase.

**Pull request merged**